staging/src/k8s.io/code-generator/cmd/informer-gen/generators/generic.go

```diff
@@ -162,14 +163,14 @@ func (f *genericInformer) Lister() {{.cacheGenericLister|raw}} {
 var forResource = `
 // ForResource gives generic access to a shared informer of the matching type
 // TODO extend this to unknown resources with a client pool
-func (f *sharedInformerFactory) ForResource(resource {{.schemaGroupVersionResource|raw}}) (GenericInformer, error) {
+func (f *sharedInformerFactory) ForResource(ctx {{.context|raw}}, resource {{.schemaGroupVersionResource|raw}}) (GenericInformer, error) {
```

Comment:
I'm trying to remember, what was the purpose of a context object on a shared informer that makes repeated, long-lived calls? deadline and timeout options aren't likely to be useful